### ISSUE TITLE
nullpointer in internal gps - Blackberry Tour

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/openwig#56]
```
What cartridge are you playing?
I haven't got that far yet.  I installed it and it ran fine when I setup 
my GPS, I closed it out and launched it again and received the error" 
Java.lang.NullPointerExecption (null) at internal

What is your phone/PDA manufacturer and model? What GPS are you using?
Blackberry Tour / Internal GPS

What steps will reproduce the problem?
1. Just launch the program
2.
3.


What is the expected output? What happens instead?
It beeps and gives the above error message

If you see an error screen, please type its contents here:
Java.lang.NullPointerExecption (null) at internal


What version are you using?
Experimental version 0.3.91

Please provide any additional information below.

```

Original issue reported on code.google.com by `l8nighter` on 26 Aug 2009 at 8:44
- Merged into: #41

**Comment by GoogleCodeExporter** [comment]
```
i believe that this is because your gps is turned off when you start OpenWIG. i
promised to fix this problem some time ago, but never actually got to it ;e) 
for now,
it should be sufficient to turn on the gps before starting the application.
```

Original comment by `matej...@gmail.com` on 26 Aug 2009 at 9:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
please let me know if that fixes the problem - if yes, then this is a duplicate 
of
issue41
```

Original comment by `matej...@gmail.com` on 26 Aug 2009 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the quick reply.  Because my GPS is internal to my Blackberry are 
you 
saying I should launch say Google Maps and have it get a lock then close out 
and 
startup openWIG and see if I get the same message?
```

Original comment by `l8nighter` on 26 Aug 2009 at 9:28

**Comment by GoogleCodeExporter** [comment]
```
maybe...
i'm told that blackberries have a feature that lets you turn the internal gps 
on or
off completely - see
http://forums.groundspeak.com/GC/index.php?showtopic=207358&st=50&p=3852204&#ent
ry3852204
```

Original comment by `matej...@gmail.com` on 26 Aug 2009 at 9:38

**Comment by GoogleCodeExporter** [comment]
```
I get this same error on the Blackberry Storm.  The internal GPS doesn't seem 
to work
with openWIG but it does work with other apps (ie: BlackStar, Trimble Navigator,
Cacheberry, Trapster, etc..)
```

Original comment by `hans.f.f...@gmail.com` on 13 Nov 2009 at 6:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `matej...@gmail.com` on 9 Feb 2010 at 4:29
- Changed title: **nullpointer in internal gps - Blackberry Tour**
- Added labels: **Type-Defect**, **Component-GPS**, **Phone-BlackBerry**

**Comment by GoogleCodeExporter** [comment]
```
let's track this in that other bug
```

Original comment by `matej...@gmail.com` on 16 Jun 2010 at 8:16
- Changed state: **Duplicate**
